packages/@dcl/ecs/src/engine/input.ts

```diff
@@ -201,11 +207,8 @@ export function createInputSystem(engine: IEngine): IInputSystem {
     pointerEventType: PointerEventType,
     entity?: Entity
   ): PBPointerEventsResult_PointerCommand | null {
-    const component = PointerEventsResult.getOrNull(engine.RootEntity)
-    if (!component) return null
-
     // We search the last pointer Event command sorted by timestamp
-    const command = findLastAction(component.commands, pointerEventType, inputAction, entity)
+    const command = findLastAction(Array.from(commandIterator()), pointerEventType, inputAction, entity)
```

Comment:
It would be nice for `findLastAction` to receive an enumerator to prevent re-allocating and re-iterating the `commandIterator()`